samples/meetings-content-bubble/csharp/ContentBubble/Bots/ContentBubbleBot.cs

```diff
@@ -76,23 +137,78 @@ private async Task HandleActions(ITurnContext<IMessageActivity> turnContext)
                         Notification = new
                         {
                             AlertInMeeting = true,
-                            ExternalResourceUrl = $"https://teams.microsoft.com/l/bubble/{ _config["MicrosoftAppId"] }?url=" +
+                            ExternalResourceUrl = $"https://teams.microsoft.com/l/bubble/{_config["MicrosoftAppId"]}?url=" +
                                                   HttpUtility.UrlEncode($"{_config["BaseUrl"]}/ContentBubble?topic={agendaItem.Topic}") +
                                                   $"&height=270&width=250&title=ContentBubble&completionBotId={_config["MicrosoftAppId"]}"
                         }
                     };
                     await turnContext.SendActivityAsync(activity);
                     break;
                 case "SubmitFeedback":
-                    var submitFeedback = Newtonsoft.Json.JsonConvert.DeserializeObject<SubmitFeedbackAction>(turnContext.Activity.Value.ToString());
+                    var submitFeedback = JsonConvert.DeserializeObject<SubmitFeedbackAction>(turnContext.Activity.Value.ToString());
                     var item = _agenda.AgendaItems.First(a => a.Id.ToString() == submitFeedback.Choice);
                     await turnContext.SendActivityAsync($"{turnContext.Activity.From.Name} voted **{submitFeedback.Feedback}** for '{item.Topic}'");
                     break;
+                case "SendNotification":
```

Comment:
How about we updating the command name to be "SendTargetedMeetingNotification" or something that contains "targeted"?

Reply:
Yes updated it